pages/learn/develop/cloud-iot-provisioning.md

```diff
@@ -42,9 +46,9 @@ The provisioning tools set up the {{ $cloud.cloudFunctionName }} function itself
 
 * configuration of the tools for testing and deployment
 * testing the function locally
-* deployment to {{ $cloud.shortName }} and end-to-end testing
+* deployment to {{ $cloud.platformName }} and end-to-end testing
```

Comment:
These two instances are the reason for adding `platformName` to the cloud dictionary.